python/ThirteenTeV/Hadronizer/Hadronizer_TuneCP5_13TeV_EEFilter_LHE_pythia8_cff.py

```diff
@@ -0,0 +1,28 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.2017MCTunes.PythiaCP5Settings_cfi import *
```

Comment:
@bendavid these maps https://github.com/cms-sw/cmssw/blob/CMSSW_9_3_0/Configuration/Generator/python/2017MCTunes/PythiaCP5Settings_cfi.py so should be good (unless python doesn't like words starting with a number instead of letter...)

Reply:
indeed python doesn't like this... there are still ways to import but they are less clean.
PR to fix in 93x and master: https://github.com/cms-sw/cmssw/pull/20571 https://github.com/cms-sw/cmssw/pull/20572